ynr/apps/elections/templates/elections/party_for_ballot.html

```diff
@@ -37,7 +37,7 @@ <h1>
                 {% if DATE_TODAY >= election.election_date %}
                   {% if user_can_record_results and show_confirm_result and not candidate_elected %}
                     {% if 'local' not in election.slug %}
-                      <form class="winner-confirm" action="{% url 'record-winner' election=election.slug post_id=post_data.id %}" method="post">
+                      <form class="winner-confirm" action="{% url '(?P<election>[^/]+)' election=election.slug post_id=post_data.id %}" method="post">
```

Comment:
Just checking this is correct, as it is different to the other url changes?

Reply:
Good spot, this must have been a find and replace error!